ui/app/routes/vault/cluster/clients/history.js

```diff
@@ -2,6 +2,7 @@ import Route from '@ember/routing/route';
 import { isSameMonth } from 'date-fns';
 import RSVP from 'rsvp';
 import getStorage from 'vault/lib/token-storage';
+import { parseRFC3339 } from '../../../../utils/date-formatters';
```

Comment:
Rather than moving up the tree you could use the vault alias for this import:
`import { parseRFC3339 } from 'vault/utils/date-formatters';`

Reply:
Ah yes - thank you! It auto-imported like this and I didn't even think to clean it up!